reports/Oregon.md

```diff
@@ -32,6 +33,7 @@ Police shoot about a dozen teargas or flashbang grenades into a crowd from quite
 **Links**
 
 * https://twitter.com/matcha_chai/status/1268043556913987584
+* https://www.oregonlive.com/portland/2020/06/6th-night-of-protests-over-george-floyd-killing-underway-in-downtown-se-portland-live-updates.html
```

Comment:
This article doesn't seem to be specific to this incident rather it covers a range of protest actions and police incidents from that day. If there are useful details in the text please copy them directly above.

Reply:
Upon second review, the article isn't 100% pellucid on whether this is the correct location. Likely, but could be wrong. I'd close for now and I'll keep an eye out for alternate sources.

Reply:
Ok awesome